Chatkit/Chatkit/Chatkit.swift

```diff
@@ -289,6 +289,26 @@ public class Chatkit {
         }
     }
     
+    /// Retrieves a static snapshot of the `User`s who are currently members of the `Room`.
+    ///
+    /// - Parameters:
+    ///     - `room`: The `Room` for which members should be retrieved.
+    public func members(for room: Room) -> [User] {
+        let roomManagedObjectID = room.objectID
+        
+        let context = persistenceController.mainContext
+        let predicate = NSPredicate(format: "ANY %K == %@", #keyPath(UserEntity.room), roomManagedObjectID)
+        let sortDescriptor = NSSortDescriptor(key: #keyPath(UserEntity.identifier), ascending: true) { (lhs, rhs) -> ComparisonResult in
```

Comment:
At some point I will improve this numeric search. It is used in many places in the code, but that does not concern this PR.